leveldbjni/src/main/java/org/fusesource/leveldbjni/internal/NativeDB.java

```diff
@@ -162,12 +162,6 @@ static final native void CompactRange(
                 @JniArg(flags={NO_OUT}) NativeSlice begin,
                 @JniArg(flags={NO_OUT}) NativeSlice end
                 );
-
```

Comment:
I'm not sure if we need to comment out below methods. The idea is that we want to keep the changes to minimal.

Reply:
These annotated methods are used by hawtjni to create the mapping. Since these are not implemented by the C++ codebase, hawtjni errors would be produced.
`[INFO] src/leveldbjni.cpp:426:35: error: 'class leveldb::DB' has no member named 'ResumeCompactions'
[INFO]   ((leveldb::DB *)(intptr_t)arg0)->ResumeCompactions();
`